webgl.js

```diff
@@ -874,6 +866,9 @@ function createANGLEInstancedArrays (context) {
   return result
 }
 
+var _checkAngleInstancedArraysSupport = gl.checkAngleInstancedArraysSupport
```

Comment:
Does this need to be implemented in native code? Could the original `gl.getSupportedExtensions` (see latest code in master) be queried to return support for angleinstancedarrays?

For example, instead of calling `_checkAngleInstancedArraysSupport`, perhaps `_getSupportedExtensions` could be called, and the presence of `"GL_ANGLE_instanced_arrays"` could be checked.

This would simplify the changes in `webgl.cc`